scripts/zIndex.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
![6f4](https://user-images.githubusercontent.com/2802867/162214495-3f4e8158-493d-46c6-beeb-2a6f85d74f57.jpeg)

Reply:
oups that shouldn't be in that PR haha

Reply:
oups that shouldn't be in that PR haha